packages/vulcan-ui-material/lib/components/core/Datatable.jsx

```diff
@@ -44,7 +44,8 @@ const baseStyles = theme => ({
     alignItems: 'center',
   },
   scroller: {
-    overflow: 'auto',
+		overflowX: 'auto',
+		overflowY: 'hidden'
```

Comment:
Not sure this work, there is a known issue with overflows which I can't remember (like if you get overflowX you can't hide  overflowY but maybe I am wrong)

Reply:
An issue in browsers themselves so we can't do anything about it